src/containers/User/UserArea.tsx

```diff
@@ -1,81 +1,76 @@
-              {currentUser?.organisation?.orgName || ''}
-              <Icon size="small" name="angle down" />
-            </div>
-          </div>
-        )}
+    <Container id="user-area">
+      <div id="user-area-left">
+        <MainMenuBar />
+        {currentUser?.organisation?.orgName && <OrgSelector user={currentUser} />}
       </div>
+      <UserMenu user={currentUser as User} />
+    </Container>
+  )
+}
+
+const MainMenuBar: React.FC = () => {
+  // TO-DO: Logic for deducing what should show in menu bar
+  // Probably passed in as props
+  return (
+    <div id="menu-bar">
+      <List horizontal>
+        <List.Item>
+          <Link to="/" className="selected-link">
+            {/* <Icon name="home" /> */}
+            {strings.MENU_ITEM_DASHBOARD}
+          </Link>
+        </List.Item>
+        <List.Item>
+          <Link to="/">Menu Item 1</Link>
+        </List.Item>
+        <List.Item>
+          <Link to="/">Menu Item 2</Link>
+        </List.Item>
+      </List>
+    </div>
+  )
+}
+
+const OrgSelector: React.FC<{ user: User }> = ({ user }) => {
+  // TO-DO: Make into Dropdown so Org can be selected
+  return (
+    <div id="org-selector">
+      {/* TO-DO: Replace with proper company logo */}
+      <Image src="/images/temp_logo.png" />
       <div>
-        <Button animated style={inlineStyles.user} onClick={() => logout()}>
-          <Button.Content visible>
-            {currentUser?.firstName || ''} {currentUser?.lastName || ''}
-          </Button.Content>
-          <Button.Content hidden>
-            <Icon name="log out" />
-          </Button.Content>
-        </Button>
+        {user?.organisation?.orgName || ''}
+        <Icon size="small" name="angle down" />
       </div>
-    </Container>
+    </div>
   )
 }
 
-// Styles - TODO: Move to LESS || Global class style (semantic)
-const inlineStyles = {
-  container: {
-    position: 'fixed',
-    display: 'flex',
-    background: '#4A4A4A',
-    top: 0,
-    zIndex: 1000,
-    height: 135,
-    padding: 20,
-    justifyContent: 'space-between',
-  } as CSSProperties,
-  top: { display: 'flex', flexDirection: 'column' } as CSSProperties,
-  link: { color: 'rgb(240,240,240)', fontSize: 14, letterSpacing: 1 } as CSSProperties,
-  left: { marginTop: 10, display: 'flex', alignItems: 'center' } as CSSProperties,
-  company: {
-    marginLeft: 20,
-    color: 'rgb(200,200,200)',
-    fontSize: 27,
-    fontWeight: 500,
-    letterSpacing: 1.2,
-    textTransform: 'uppercase',
-  } as CSSProperties,
-  image: { height: 60, width: 60, boxShadow: '0px 0px 3px rgb(240,240,240)' } as CSSProperties,
-  user: {
-    background: 'rgb(248,248,248)',
-    border: 'none',
-    borderRadius: 20,
-    fontSize: 14,
-    color: 'rgb(50,50,50)',
-    paddingRight: 5,
-  } as CSSProperties,
+const UserMenu: React.FC<{ user: User }> = ({ user }) => {
+  const { logout } = useUserState()
+  return (
+    <div id="user-area-user-menu">
```

Comment:
I was wondering is this `id` should be called  `user-area-right` for consistency?

Reply:
I only called the other side "left" because it was itself a container for two other elements, whereas this one is just the element itself, there's nothing more specific inside it. How about I just shorten it to "user-menu"?

Reply:
Yes, agreed